### PR TITLE
rule maintenance - 3

### PR DESCRIPTION
**Pull request by SKEIDs**
adjust duplicate rules.

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -29,7 +29,6 @@ kakuyomu.jp###readWithApp
 kakuyomu.jp##.widget-dynamicLinksButton
 1010dy.cc##.col-xs-1 > div > a[href="http://www.shuangshi.cc/"][target="_blank"] > img
 uteka.ru##.app-discount-mini-banner
-carnegie.ru##.section--2x + strong.letter-space
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt:32]: Why removed? It is used here: https://carnegie.ru/?lang=ru

**Review by Alex-302 — APPROVED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/mobile-app.txt:32]: Merged PR and restored this rule.

Comment (Alex-302) [reply AnnoyancesFilter/sections/mobile-app.txt:32]: And thanks)

Comment (SKEIDs) [reply AnnoyancesFilter/sections/mobile-app.txt:32]: I removed that by mistake. sorry :(

**Pull request merged**